File: API.RBS/Models/Customer.cs
using System;
using System.Collections.Generic;

namespace API.RBS.Models
{
    public class Customer : User
    {
        public string Phone { get; set; }
        public string Email { get; set; }
        public string Address { get; set; }
        public DateTime DateOfBirth { get; set; }
        public string Gender { get; set; }
        public int Height { get; set; }
        public int Weight { get; set; }
        public string Purpose { get; set; }
        public int? InstructorId { get; set; }
        public Instructor Instructor { get; set; }
        public ICollection<Symptom> Symptoms { get; set; }
    }
}

File: SPA-RBS/src/app/_models/customer.ts
import { Symptom } from './symptom';
import { User } from './user';
import { Programme } from './programme';
import { CustomerImage } from './customerImage';

export interface Customer extends User {
    phone: string;
    email: string;
    address: string;
    age: number;
    gender: string;
    height: string;
    weight: string;
    purpose: string;
    symptoms: Symptom[];
    programmes: Programme[];
    customerImages: CustomerImage[];
}
